### PR TITLE
Better Docs 

### PR DESCRIPTION
**Pull request by lidevelopers**
Better instructions about deploying the bot to heroku

### CODE REVIEW

**Review by OIVAS7572 — CHANGES_REQUESTED** [review]
Don't enable Automatic Deploys. Some Big files are downloading while deploying. Some times It will not work because of Over using resources.
Branch will be master not main.

**Comment by lidevelopers** [comment]
now is it ok?

File: README.md [hunk 1]
```diff
@@ -10,13 +10,15 @@
...
+- Edit only your token in the `config.yml` file over [here](/config.yml#L1).
+- Install [Heroku CLI](https://devcenter.heroku.com/articles/heroku-cli) and Create a [new heroku app](https://dashboard.heroku.com/new-app).
 **Do note that in certain operating systems Heroku CLI doesn't get added to path automatically. If that's the case you'll have to add heroku to your path manually.**
 - Run this in cmd `heroku stack:set container -a appname` replace appname with your Heroku app's name.
-- Connect app with Github and deploy app in Heroku.
-- After deploying turn worker on in Resources menu in Heroku. 
+- Click on `search` and then select your fork of this repository.
+- Go to the `Deploy` tab and click `Connect to GitHub`.
+- Once it has been deployed, go to `Resources` tab on heroku and enable `worker (python3 lichess-bot.py -u)` dynos. (Do note that if you don't see any dynos in the `Resources` tab, then you must wait for about 5 minutes and then refresh your heroku page.)
+- You're now connected to lichess and awaiting challenges! Your bot is up and ready!
```
Comment (OIVAS7572) [inline README.md:21]: what you mean by 21 st line . This is showed in logs

**Comment by OIVAS7572** [comment]
@lidevelopers 
what you mean by 21 st line . This is showed in logs

**Review by OIVAS7572 — APPROVED** [review]

**Comment by lidevelopers** [comment]
Are you merging

**Comment by TheYoBots** [comment]
I think it skreens up the read me a bit. Will have to look at something neater

**Comment by OIVAS7572** [comment]
Some of the changes by @lidevelopers  is Good

**Pull request closed without merging**